src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/CreateCaseTest.java

```diff
@@ -92,6 +94,46 @@ public void should_create_case_from_valid_exception_record() throws Exception {
         assertThat(createdCase.getData().get("email")).isEqualTo("hello@test.com");
     }
 
+    @Test
+    @Disabled
+    public void should_idempotently_create_case_from_valid_exception_record() throws Exception {
+        // given
+        CaseDetails exceptionRecord = createExceptionRecord("envelopes/new-envelope-create-case-with-evidence.json");
+
+        // when
+        // create case callback endpoint invoked first time
+        AboutToStartOrSubmitCallbackResponse callbackResponse = invokeCallbackEndpoint(exceptionRecord);
+        String caseCcdId = getCaseCcdId(callbackResponse);
+
+        // then
+        CaseDetails createdCase = ccdApi.getCase(caseCcdId, exceptionRecord.getJurisdiction());
+        assertThat(createdCase.getCaseTypeId()).isEqualTo(BULK_SCANNED_CASE_TYPE);
+        assertThat(createdCase.getData().get("firstName")).isEqualTo("value1");
+        assertThat(createdCase.getData().get("lastName")).isEqualTo("value2");
+        assertThat(createdCase.getData().get("email")).isEqualTo("hello@test.com");
+
+        assertThat(createdCase.getData().get("bulkScanCaseReference")).isNotNull();
+        String bulkScanCaseReference = (String)createdCase.getData().get("bulkScanCaseReference");
+        assertThat(bulkScanCaseReference.equals(exceptionRecord.getId()));
+
+        await("Case is ingested")
+            .atMost(2, TimeUnit.SECONDS)
+            .pollDelay(1, TimeUnit.SECONDS)
+            .until(() -> caseIngested(bulkScanCaseReference));
+
+        // when
+        // create case callback endpoint invoked second time
+        AboutToStartOrSubmitCallbackResponse callbackResponse2 = invokeCallbackEndpoint(exceptionRecord);
+        String caseCcdId2 = getCaseCcdId(callbackResponse2);
```

Comment:
can we do all setup in one go?
ie:
1. send request for the 1st time,
2. send request for the 2nd time,
3. make assertions

Reply:
The idea here is to replicate the test above (should_create_case_from_valid_exception_record) and add the check for idempotency. The test above will be deleted once this test starts passing.